scripts/Editor/DllManipulatorEditor.cs

```diff
@@ -210,8 +266,6 @@ private void DrawOptions(DllManipulatorOptions options)
         {
             var guiEnabledStack = new Stack<bool>();
             guiEnabledStack.Push(GUI.enabled);
```

Comment:
If you remove this you could also remove the stack thing, this is the push here and corresponding pop below.